TMSFTT/urls.py

```diff
@@ -19,6 +19,7 @@
 
 
 API_URLPATTERNS = [
+    path('training-record', include('training_record.urls')),
```

Comment:
While this is OK, it's better to keep this close to other training* routes, so please move it down.